pkg/machines/components/createDialogUtils.jsx

```diff
@@ -0,0 +1,233 @@
+/*jshint esversion: 6 */
```

Comment:
I can't see anything requiring .jsx extension in this file. Can we rename to .es6?